### ISSUE TITLE
Not support flat dictionary

### ISSUE THREAD

**Issue by heidarbozorg** [Tornhoof/SpanJson#197]
I've just tried to test _**Dictionary**_ type and _**[JsonExtensionData]**_ attribute for making flat JSON serialized. It seems not to work properly.
Here is the simple model:
```
public class Simple
{
    public string Key;
    public string Value;

    [JsonExtensionData]
    public Dictionary<string, object> AdditionalValues { get; set; }

    public Simple()
    {
        Key = "KeyValue";
        Value = "ValueValue";
        AdditionalValues = new Dictionary<string, object>()
        {
            { "intKey", 123 },
            { "stringKey", "Some string value" },
            { "doubleKey", 1.24 },
            { "nullKey", null }
        };
    }
}
```

The JSON result is:
`{"Key":"KeyValue","Value":"ValueValue","AdditionalValues":{"intKey":123,"stringKey":"Some string value","doubleKey":1.24,"nullKey":null}}`

I expected it should be:
`{"Key":"KeyValue","Value":"ValueValue","intKey":123,"stringKey":"Some string value","doubleKey":1.24,"nullKey":null}`

**Comment by Tornhoof** [comment]
Are you sure you're using it correctly, e.g. that this is actually the SpanJson attribute and not the one from json.net or another library?

Please provide a full standalone repro.

**Comment by heidarbozorg** [comment]
Oh, my bad!
Sorry, you are right. I used the `[JsonExtensionData]` attribute under `System.Text.Json.Serialization` namespace. 
Now it works fine by using `[SpanJson.JsonExtensionData]` attribute.

Just suggest you to bold this in your documentation. Since `[JsonExtensionData]` attribute is already around, by default I used the  ` [JsonExtensionData]` attribute under `System.Text.Json.Serialization`.

Thank you for your support.
